### PR TITLE
Add blurbs for freeze-spec-const pass

### PR DESCRIPTION
**Pull request by Qining**
Add back the missing blurbs of pass: freeze-spec-consts

### CODE REVIEW

File: source/opt/passes.h [hunk 1]
```diff
@@ -58,6 +58,14 @@ class StripDebugInfoPass : public Pass {
   bool Process(ir::Module* module) override;
 };
 
+// The transformation pass that specialize the value of spec constants to their
```
Comment (dneto0) [inline source/opt/passes.h]: "specializes"

File: source/opt/passes.h [hunk 2]
```diff
@@ -58,6 +58,14 @@ class StripDebugInfoPass : public Pass {
...
+// default values. This pass only process the spec constants that have Spec ID
```
Comment (dneto0) [inline source/opt/passes.h]: "processes"

File: source/opt/passes.h [hunk 3]
```diff
@@ -58,6 +58,14 @@ class StripDebugInfoPass : public Pass {
...
+// decorations (defined by OpSpecConstant, OpSpecConstantTrue and
+// OpSpecConstantFalse instructions) and replace them with their front-end
```
Comment (dneto0) [inline source/opt/passes.h]: "replaces"

**Comment by dneto0** [comment]
-1
Just singular-plural grammar stuff.

**Comment by antiagainst** [comment]
+2

**Pull request merged**